app/views/receptacles/new_request.html.erb

```diff
@@ -9,7 +9,7 @@
     <% instance_variable_or_id_param(:request_type) do |field_name, value| %>
       <div class="form-group  col-xs-12">
         <%= label_tag(field_name, 'Request type', class: 'col-md-4') %>
```

Comment:
Optional but it might be good to add the 'required' class here as that adds a star indicator to the users that the label is required.

Reply:
Thank you Ben, I don't think they are really required fields, but also I suppose it depends of the type of the request.

Reply:
It says in the story request type is required:
_The request type box is mandatory for all requests, and sequencing cannot be requested without this being selected_

Reply:
as are Read length and Flow cell type